internal/move2kube/planner.go

```diff
@@ -76,6 +78,15 @@ func CreatePlan(inputPath string, prjName string, interactive bool) plantypes.Pl
 	}
 	log.Infoln("Translation planning done")
 
+	// sort the service options in order of priority
+	for serviceName, serviceOptions := range p.Spec.Inputs.Services {
+		log.Debugf("Before sorting options of service: %s service options:\n%v", serviceName, serviceOptions)
+		sort.Slice(serviceOptions, func(i, j int) bool {
+			return containerizer.ComesBefore(serviceOptions[i].ContainerBuildType, serviceOptions[j].ContainerBuildType)
```

Comment:
Should we also consider other attributes for sorting, say translation type, folder level of sourceartifacts folder etc?

Reply:
why?